### DISCUSSION TITLE
Lost tables after upgrade to 7.3.2

### DISCUSSION THREAD

**Discussion by hardyb** [dbeaver/dbeaver#11134] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version = Win 10 Pro x64
- DBeaver version = 7.3.2
- Additional extensions = None

#### Connection specification:
- Database name and version = postgres
- Driver name = PostresSQL
- Do you use tunnels or proxies (SSH, SOCKS, etc)? = No

#### Describe the problem you're observing:
Last time I used DBeaver I had just finished creating a new database with a significatnly large number of tables, relationships, various constraints, functions, etc, many hours of work - all in Database Navigator under
PostresSQL localhost / 
When I next started DBeaver a modal dialog opened asking me to upgrade - (I think I was currently on 6.something) I selected to upgrade and was automatically taken to the dbeaver community web page and a download began immediately.  The download was for 7.3.2 , I installed the upgrade and reopened DBeaver.

After re-opening DBeaver I was surprised to see under the (auto-reconnected) same connection I had just been using, PostresSQL localhost / 
Only the postgres (default) database, nothing else.

For a bit I thought the other databases were lost - other db management UI on re-connecting the same named connection (after upgrade), with the same properties you usually get all the same databases.

After some fiddling about, I discovered that the 'see all databases' setting on the connection must have been reset to unchecked.
Is this kind of (possibly unexpected, possibly disconcerting) behaviour following an upgrade intended? or is it a bug?

Thanks

**Comment by kseniiaguzeeva** [comment, marked as answer]
Sorry, that you spent a time to find the reason. Some settings were changed in previous DBeaver versions and Show all databases was disabled by default. We try to avoid such changes.
